app/services/g6importService.py

```diff
@@ -0,0 +1,62 @@
+import json
```

Comment:
Naming conventions in python tend towards `snake_case` rather than `camelCase`. For variable names this is just a convention for module names this is prescribed for module names. See: https://www.python.org/dev/peps/pep-0008/#prescriptive-naming-conventions